Questions 5, 8, 13, 18

Nika-Chemerys opened the pull request:
Question 5: A function that returns the first name and the last name reversed and with a space between them
Question 8: A function that calculates the number of days from the birthday
Question 13: A function that converts length in cm to inches
Question 18: A function that reverse the tuple

Nika-Chemerys committed:
Questions 5, 8, 13, 18

Question 5: A function that returns the first name and the last name reversed and with a space between them
Question 8: A function that calculates the number of days from the birthday
Question 13: A function that converts length in cm to inches
Question 18: A function that reverse the tuple

veronikachyzhykova commented:
Всі програми виконують задані умови, супер! 
Але маю зауваження щодо завдання 13:
функція нічого не повертає, тому маємо зайвий радок "None".
Спробуй замінити print на return

Nika-Chemerys committed:
Questions 5, 8, 13, 18

Question 5: A function that returns the first name and the last name reversed and with a space between them
Question 8: A function that calculates the number of days from the birthday
Question 13: A function that converts length in cm to inches
Question 18: A function that reverse the tuple

Nika-Chemerys committed:
Delete Homework_15_09_2022 (1).ipynb
